.github/workflows/ci-build.yml

```diff
@@ -148,12 +150,12 @@ jobs:
       continue-on-error: true
 
     - name: Code coverage
-      env:
-        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
-        COVERALLS_GIT_COMMIT: ${{ github.sha }}
-        COVERALLS_GIT_BRANCH: ${{ github.ref }}
       run: |
         ./vendor/bin/phpunit --coverage-clover /tmp/coverage/clover_executor.xml
         EXECUTOR=coroutine ./vendor/bin/phpunit --coverage-clover /tmp/coverage/clover_executor-coroutine.xml
-        wget https://github.com/php-coveralls/php-coveralls/releases/download/v2.2.0/php-coveralls.phar
-        php7.1 php-coveralls.phar --verbose
+
+    - name: Report to Coveralls
+      env:
+        COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COVERALLS_RUN_LOCALLY: 1
```

Comment:
If you delete this row, script in section run doesn't work?

Reply:
I check here https://github.com/Smolevich/reindexer-client/runs/380126623, script fails

Reply:
:) I was fine with it working for now and did not want to investigate it further but it's something that should be IMO resolved in https://github.com/php-coveralls/php-coveralls/pull/275 before merging.